### ISSUE TITLE
Incident collected through crashlytics

### ISSUE THREAD

**Issue by adrianddpp** [Aghajari/ZoomHelper#8]
Hi!
First of all, thank you very much for this library. I attach the report of a crash produced by this library, I think it can be useful for me and more people.

Android 9
Redmi 5 Plus

> Fatal Exception: java.lang.NullPointerException
       at com.aghajari.zoomhelper.ZoomHelper$dismissDialogAndViews$1.run(ZoomHelper.kt:351)
       at android.os.Handler.handleCallback(Handler.java:873)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:201)
       at android.app.ActivityThread.main(ActivityThread.java:6810)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)

If you could help me to solve this I would appreciate it, thanks

**Comment by Aghajari** [comment]
Hi, 
Which version of the lib are you using?
i believe the last version works fine!

**Comment by adrianddpp** [comment]
Hello, thanks for answering first of all.
The version I have installed is 1.0.1, I have seen that there is a higher version, I am going to apply this update to test if the error goes away :)

It seems that the problem occurs on this line ZoomHelper.kt:351

> val parent = zoomableView!!.parent as ViewGroup;

Maybe it could be interesting to control that zoomableView is a viewGroup, this could be an alternative, and that is not null

```
(zoomableView!!.parent as? ViewGroup)?.let{ parent->
  parent.removeView(zoomableView)
                    zoomableViewParent?.addView(zoomableView!!, viewIndex!!, viewLayoutParams)
                    zoomableViewParent?.removeView(placeHolderView)
                    dismissDialog()
}
```

It seems that since I added this fire I have received more reports. I could upload a pull request if you see it interesting.

Regards!
